### DISCUSSION TITLE
easywechat和PHP版本函数不匹配

### DISCUSSION THREAD

**Discussion by xinlinnet** [w7corp/easywechat#2407] [General]
easywechat php版本要求>8.0.2。但是使用了php8.1的函数array_is_list。导致如果安装的是php8.0的话是无法正常运行的。undefined function array_is_list()

**Comment by overtrue** [comment]
https://github.com/w7corp/easywechat/releases/tag/6.2.2 

已修复，感谢反馈！

**Comment by senmeng** [comment]
怎么处理这个问题，怎么更新

**Reply by xinlinnet** [reply]
看上面作者的回复。或者升级PHP版本到8.1
